Xamarin.Forms.Platform.Android/Renderers/EntryRenderer.cs

```diff
@@ -477,4 +485,85 @@ protected virtual void UpdateIsReadOnly()
 			Control.Focusable = isReadOnly;
 		}
 	}
+
+	// Entry clear button management
+	public abstract partial class EntryRendererBase<TControl>
```

Comment:
I'm not decided. I initially wanted to have the code which deals with clear button separately. 

Using a static class like `EntryClearButtonManager` could be better but I wanted to have a way to override the drawable in case someone wants to: 
`protected virtual Drawable GetCloseButtonDrawable()`.
 Another option is to have this virtual method in the EntryRendererBase class and pass the drawable to the `EntryClearButtonManager`.

Or I could just give up on this idea and put everything int the EntryRendererBase..
What do you think?

Reply:
The manager might be a good idea. You can also use an interface so that the manager can talk back to the entry.

Reply:
I can do that, sure. Yet, the partial class is simple and it doesn't need additional classes/interfaces... 
I wonder if it would help if I just move it to a separate file like EntryBaseRenderer.ClearButton.cs

Reply:
looks like a strange way of not doing `#region`s